networking/p2p/src/test/java/tech/pegasys/teku/networking/p2p/DiscoveryNetworkTest.java

```diff
@@ -88,7 +91,7 @@ public void shouldStopConnectionManagerBeforeNetworkAndDiscovery() {
     discoveryNetwork.stop();
 
     verify(connectionManager).stop();
-    verifyNoInteractions(p2pNetwork, discoveryService);
```

Comment:
May be worth verifying the expected `discoveryService` interaction, then asserting `verifyNoMoreInteractions(discoveryService)`.

Reply:
good idea!